Components/SettingsBar/SettingsBar.tsx

```diff
@@ -0,0 +1,74 @@
+import { useRouter } from "next/router";
```

Comment:
we can just keep the file name to `Components/SettingsBar/index.tsx`

Reply:
Ok sir